tools/tap-driver

```diff
@@ -132,7 +133,10 @@ class Driver:
             self.report("SKIP", *args)
 
     def report_error(self, description=""):
-        self.report("ERROR", "", description)
+        if self.enable_hard_errors:
+            self.report("ERROR", "", description)
+        else:
+            self.result_fail(description)
```

Comment:
The default of that option is `False`, so that changes behaviour. What's the visible effect of that?

Reply:
This enables the use of [XFAIL_TESTS](https://gitlab.gnome.org/GNOME/libsecret/blob/master/.gitlab-ci.yml#L36) when the test suite runs under automake's harness.